lib/dscli/source.rb

```diff
@@ -11,100 +11,74 @@ def list(page = 1)
-      puts response[:data].to_yaml
-    rescue ApiResourceNotFoundError => e
-      puts "Specified managed source '#{id}' was not found. It may have been deleted."
-    end
-
+    api = Dscli::API.new
+    response = api.source_get(id)
+    puts response[:data].to_yaml
+  rescue ApiResourceNotFoundError
+    puts "Specified managed source '#{id}' was not found. It may have been deleted."
   end
 
   desc 'start (id)', 'Starts the specified source'
   def start(id)
-
   	api = Dscli::API.new
-      
-    begin
-      response = api.source_start(id)
+    response = api.source_start(id)
 
-      if response[:http][:status] == 200
-        puts "Managed source '#{id}' started successfully"
-      else
-        # TODO: How do we handle a different code?
-        response
-      end
-    rescue ApiResourceNotFoundError => e
-      puts "Specified source '#{id}' not found. It may have been deleted."
+    if response[:http][:status] == 200
+      puts "Managed source '#{id}' started successfully"
+    else
+      # TODO: How do we handle a different code?
+      response
     end
-
+  rescue ApiResourceNotFoundError
+    puts "Specified source '#{id}' not found. It may have been deleted."
   end
 
   desc 'stop (id)', 'Stops the specified source'
   def stop(id)
-
   	api = Dscli::API.new
-      
-    begin
-      response = api.source_stop(id)
+    response = api.source_stop(id)
 
-      if response[:http][:status] == 200
-        puts "Managed source '#{id}' stopped successfully"
-      else
-        # TODO: How do we handle a different code?
-        response
-      end
-    rescue ApiResourceNotFoundError => e
-      puts "Specified source '#{id}' not found. It may have been deleted."
+    if response[:http][:status] == 200
+      puts "Managed source '#{id}' stopped successfully"
+    else
+      # TODO: How do we handle a different code?
+      response
     end
-
+  rescue ApiResourceNotFoundError
+    puts "Specified source '#{id}' not found. It may have been deleted."
   end
 
   desc 'delete (id)', 'Deletes a managed source'
   def delete(id)
     api = Dscli::API.new
+    response = api.source_delete(id)
 
-    begin
-
-      response = api.source_delete(id)
-
-      if response[:http][:status] == 204
-        puts "Source '#{id}' deleted successfully"
-      else
-        response
-      end
-    rescue ApiResourceNotFoundError => e
-      puts "Source '#{id}' not found. It may have been deleted."
+    if response[:http][:status] == 204
+      puts "Source '#{id}' deleted successfully"
+    else
+      response
     end
-
+  rescue ApiResourceNotFoundError
+    puts "Source '#{id}' not found. It may have been deleted."
   end
 
   desc 'logs (id)', 'Retrieves the log for a managed source'
   def logs(id = nil)
     api = Dscli::API.new
+   	results = api.source_log(id)
 
-    begin
+	  puts "\nMessage count: #{results[:count]}\n\n"
+	  puts ' Time                     | Message                                          '
```

Comment:
Line is too long. [87/80]<br>Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.<br>Do not write to stdout. Use Rails' logger if you want to log.